### ISSUE TITLE
Game list does not refresh when using right-click -> clear in searchbar

### ISSUE THREAD

**Issue by FaceCrap** [littleraisins/F95CheckerX#5]
When you have typed a query in the searchbar, and then use right-click -> clear, the game list does not refresh showing all entries, but keeps showing only the matched games.
It works when using Escape, or just backspace, or selecting the whole query and then pressing del (Ctrtl-A -> Delete), just not with the right-click option

**Comment by FaceCrap** [comment]
Not completely fixed. Granted, it's not a normal sequence of steps, but end-result is the same

So... I was trying to figure out how to filter on custom games other than through status. So just typed `custom` in the searchbar. It actually brought up one game, but this happened because the notes contained the actual word. Gotten curious I switch to the Reminders, only to see no game listed. Did the `right-click -> clear` and switched back to the regular games, only to find the one matching game for the query still showing, had to hit ESC again to get the full list.

So, in short, if you type a filter in one mode, and clear it after switching to the other mode, then switch back to the mode you started the filter in, that mode will still show the filtered result.

**Comment by littleraisins** [comment]
Sorry for that, I really should test "easy" things more thoroughly. Fixed now.
